yarn.lock

```diff
@@ -3,28 +3,28 @@
 
 
 "@ampproject/remapping@^2.1.0":
```

Comment:
Please revert this entire file (not sure how it got changed on you)

Reply:
done